test/e2e/basic_test.go

```diff
@@ -287,12 +292,24 @@ func TestReconcileGatewayRoutesOnStartup(t *testing.T) {
 		if err != nil {
 			return false, err
 		}
-		if len(routes) < clusterInfo.numNodes-1 {
+		expectedRtNum := clusterInfo.numNodes - 1
```

Comment:
for readability, could you use a `minNumRoutes` and a `maxNumRoutes` variables, and compute them before the `PollImmediate` call based on the encap mode?

Reply:
done.